sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_receiver.py

```diff
@@ -309,9 +340,6 @@ def receive(self, max_batch_size=None, max_wait_time=None):
 
         """
         self._check_live()
-        if max_batch_size and self._prefetch < max_batch_size:
```

Comment:
Do we need to update the documentation of the `prefetch` parameter?

Reply:
yes, I'll do the update

Reply:
updated, add extra explanation for the behavior the receive when 'prefetch' is 0